solana-programs/anchor/audius-data/programs/audius-data/src/lib.rs

```diff
@@ -596,31 +553,21 @@ pub struct RemoveUserAuthorityDelegate<'info> {
 
 /// Instruction container for track creation
```

Comment:
we can generalize this comment too

Reply:
definitely, i think we will follow up on this one